messaging/src/test/java/org/axonframework/commandhandling/SimpleCommandBusTest.java

```diff
@@ -96,7 +96,7 @@ void testDispatchCommandImplicitUnitOfWorkIsCommittedOnReturnValue() {
 
     @Test
     void testFireAndForgetUsesDefaultCallback() {
```

Comment:
I think there's merit in validating whether the span logic is correctly followed in this unit test class.